test-suite/Makefile

```diff
@@ -57,11 +56,22 @@ export CAML_LD_LIBRARY_PATH:=$(shell echo $(COQPREFIX)/lib/stublibs):$(CAML_LD_L
 
 # COQLIB is an env variable so no quotes
 COQLIB?=
+
 ifeq ($(COQLIB),)
+# This should guard againt the COQLIBINSTALL directory not existing,
+# however, it doesn't work, defying my knowledge about makefiles.
+ifneq ($(wildcard $(COQLIBINSTALL)),)
 	COQLIB := $(shell ocaml ocaml_pwd.ml $(COQLIBINSTALL))
+else
+	COQLIB := $(shell ocaml ocaml_pwd.ml $(COQLIBINSTALL))
+endif
```

Comment:
What's the point of this?

Reply:
To have someone that understands make to fix it some day.

Reply:
I'd rather remove the whole ocaml_pwd stuff, but that's a different story. The use case for the if is as follows:
- `ocaml_pwd` requires the path to exists, so if the user calls `make -C test-suite` before the path was created, and error follows.

Reply:
>To have someone that understands make to fix it some day.

But what is it supposed to do? it seems like there should be something else in the `else` branch.

Reply:
The `else` branch should be a noop, this is guarding against calling `ocaml_pwd` with a non existing path.

Better, we should get rid of `ocaml_pwd`, but that's for another day.

Reply:
I have no idea, but I was unable to have that conditional work fine.